schema-source/RS0001.schema.yaml

```diff
@@ -216,7 +216,7 @@ Performance:
     condenser_liquid_type:
       Description: "Type of liquid in condenser"
       Data Type: "{LiquidMixture}"
-      Required: "if condenser_type = LIQUID"
```

Comment:
@TimMcDowell @tanaya-mankad this is just a syntactical limitation of the legacy schema code. The legacy schema-205 code was silently misinterpreting this. In 232, we may want to build in some flexibility here.